cmd/oci-create-runtime-bundle/main.go

```diff
@@ -112,29 +102,26 @@ func (v *bundleCmd) Run(cmd *cobra.Command, args []string) {
 	logrus.SetLevel(logLevel)
 
 	if v.version {
-		v.stdout.Printf("commit: %s", gitCommit)
-		v.stdout.Printf("spec: %s", specs.Version)
+		logrus.Infof("commit: %s", gitCommit)
+		logrus.Infof("spec: %s", specs.Version)
 		os.Exit(0)
 
 	}
 	if len(args) != 2 {
-		v.stderr.Print("both src and dest must be provided")
 		if err := cmd.Usage(); err != nil {
-			v.stderr.Println(err)
+			logrus.Errorf("%s", err)
 		}
-		os.Exit(1)
+		logrus.Fatalf("both src and dest must be provided")
```

Comment:
@wking 
I've researched runtime tool repo, but found it has only debug messages, without info message(so far) need feed back to consumers.
But for image tools repo, there are messages, like https://github.com/opencontainers/image-tools/pull/46/commits/5beb0201b7ffdb41f150d6f20d8471e27bc8bd37#diff-da07f373046a0a24e67b18699dc15a70R88 and https://github.com/opencontainers/image-tools/pull/46/commits/5beb0201b7ffdb41f150d6f20d8471e27bc8bd37#diff-8c24ca9b63b3598ef3f4cc64741f433eR117, need be shown to consumers.
Later we maybe add more information, which consumer does care about, so I think default info level is better.
I'm not persistent to it, but could you please consider it? Thanks.